README.md

```diff
@@ -13,33 +13,29 @@ Your first PR should be a first draft of the final CAIP. An editor will manually
 If your CAIP requires images, the image files should be included in a subdirectory of the `assets` folder for that CAIP as follows: `assets/caip-N` (where **N** is to be replaced with the CAIP number). When linking to an image in the CAIP, use relative links such as `../assets/caip-1/image.png`.
 
 
-# CAIP Status Terms
+## CAIP Status Terms
 
 * **Draft** - an CAIP that is undergoing rapid iteration and changes.
 * **Last Call** - an CAIP that is done with its initial iteration and ready for review by a wide audience.
 * **Accepted** - a core CAIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author.
 
-# CAIP Index
+## CAIP Index
 
 * **CAIP-1** - [CAIP Purpose and Guidelines](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-1.md)
 * **CAIP-2** - [Blockchain ID Specification](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md)
-* **CAIP-3** - [Blockchain Reference for the EIP155 Namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-3.md)
-* **CAIP-4** - [Blockchain Reference for the BIP122 Namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-4.md)
-* **CAIP-5** - [Blockchain Reference for the Cosmos Namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-5.md)
-* **CAIP-6** - [Blockchain Reference for the LIP9 Namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-6.md)
-* **CAIP-7** - [Blockchain Reference for the EOSIO Namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-7.md)
+* **CAIP-6** - [Blockchain Reference for the LIP9 Namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-6.md) - see [chainagnostic/namespaces#17](https://github.com/ChainAgnostic/namespaces/pull/17)
```

Comment:
Imo we should remove CAIPs that are supposed to be in namespaces.

Reply:
Sure, but they should not be in here anyway. We can open issues for them instead of keeping them in the Readme.

Reply:
ok done let's gooo